src/headers.js

```diff
@@ -103,7 +103,7 @@ class Headers {
    * @return  Mixed
    */
   get (name) {
-    const list = this[MAP][sanitizeName(name)]
+    const list = this[MAP][name.toLowerCase()]
```

Comment:
This should probably replicate the `('' + name).toLowerCase()` behavior that `sanitizeName` does, just for consistency there.

Reply:
Also, this should be duplicated for `has()`, since that's another reader method.